<?xml version="1.0" encoding="utf-8" standalone="yes" ?>
<rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
  <channel>
    <title>Posts on HORIZON</title>
    <link>https://horizonnn.github.io/post/</link>
    <description>Recent content in Posts on HORIZON</description>
    <generator>Hugo -- gohugo.io</generator>
    <language>en-us</language>
    <lastBuildDate>Sun, 25 Aug 2019 21:21:34 +0800</lastBuildDate>
    
	<atom:link href="https://horizonnn.github.io/post/index.xml" rel="self" type="application/rss+xml" />
    
    
    <item>
      <title>BugFlow</title>
      <link>https://horizonnn.github.io/post/bugflow/</link>
      <pubDate>Sun, 25 Aug 2019 21:21:34 +0800</pubDate>
      
      <guid>https://horizonnn.github.io/post/bugflow/</guid>
      <description>基础问题 AttributeError: module &amp;lsquo;XXX&amp;rsquo; has no attribute &amp;lsquo;xxx&amp;rsquo; 原因：一般module和attribute的名字没有拼错的情况下，出现这类问题通常是因为这个py文件所在的目录中存在与module同名的py文件，导致python的错误调用。
解决办法：更改文件名。
Keras TypeError: Unexpected keyword argument passed to optimizer: name Traceback (most recent call last): File &amp;quot;visualizing_the_convnet.py&amp;quot;, line 3, in &amp;lt;module&amp;gt; model = load_model(&#39;cats_and_dogs_small_2.h5&#39;) File &amp;quot;D:\ProgramFiles\Anaconda3\envs\keras\lib\site-packages\tensorflow\python\keras\engine\saving.py&amp;quot;, line 249, in load_model optimizer_config, custom_objects=custom_objects) File &amp;quot;D:\ProgramFiles\Anaconda3\envs\keras\lib\site-packages\tensorflow\python\keras\optimizers.py&amp;quot;, line 838, in deserialize printable_module_name=&#39;optimizer&#39;) File &amp;quot;D:\ProgramFiles\Anaconda3\envs\keras\lib\site-packages\tensorflow\python\keras\utils\generic_utils.py&amp;quot;, line 194, in deserialize_keras_object return cls.from_config(cls_config) File &amp;quot;D:\ProgramFiles\Anaconda3\envs\keras\lib\site-packages\tensorflow\python\keras\optimizers.py&amp;quot;, line 159, in from_config return cls(**config) File &amp;quot;D:\ProgramFiles\Anaconda3\envs\keras\lib\site-packages\tensorflow\python\keras\optimizers.py&amp;quot;, line 247, in __init__ super(RMSprop, self).</description>
    </item>
    
  </channel>
</rss>